x/multisig/types/types.go

```diff
@@ -4,18 +4,33 @@ import (
 	"bytes"
 	"encoding/hex"
 	"fmt"
-	"sort"
 
 	"github.com/btcsuite/btcd/btcec"
-	sdk "github.com/cosmos/cosmos-sdk/types"
+)
 
-	"github.com/axelarnetwork/axelar-core/utils"
-	"github.com/axelarnetwork/axelar-core/x/multisig/exported"
-	snapshot "github.com/axelarnetwork/axelar-core/x/snapshot/exported"
-	"github.com/axelarnetwork/utils/funcs"
-	"github.com/axelarnetwork/utils/slices"
+const (
+	// HashLength is the expected length of the hash
+	HashLength = 32
 )
 
+// Hash is an alias for a 32-byte hash
+type Hash []byte
```

Comment:
```suggestion
type Hash [32]byte
```
make it fixed or is that not convenient when used?

Reply:
It won't be castable to protobuf `bytes` anymore which means then we'll have to do `customtype`, which is a bit too much.